README.md

```diff
@@ -1,110 +1,37 @@
-[![Build Status](https://circleci.com/gh/mapbox/mapbox-gl-js.svg?style=svg)](https://circleci.com/gh/mapbox/mapbox-gl-js) [![Coverage Status](https://coveralls.io/repos/github/mapbox/mapbox-gl-js/badge.svg?branch=master)](https://coveralls.io/github/mapbox/mapbox-gl-js?branch=master)
+[Mapbox GL is an ecosystem of libraries for rendering maps using WebGL/OpenGL on the web, iOS, Android, Qt, macOS and more](https://www.mapbox.com/maps/). This repository holds the files and issues shared among all platforms.
 
-# Mapbox GL JS
+[<img width="981" alt="Mapbox GL gallery" src="https://cloud.githubusercontent.com/assets/281306/14547142/a3c98294-025f-11e6-92f4-d6b0f50c8e89.png">](https://www.mapbox.com/gallery/)
 
-Mapbox GL JS is a Javascript & WebGL library that renders interactive maps from [vector tiles](https://www.mapbox.com/blog/vector-tiles/) and [Mapbox styles](https://www.mapbox.com/mapbox-gl-style-spec).
+## Mapbox GL JS
 
-It is part of the [Mapbox GL ecosystem](https://github.com/mapbox/mapbox-gl) which includes [Mapbox GL Native](https://github.com/mapbox/mapbox-gl-native), a suite of compatible SDKs for native desktop and mobile applications.
+This repository holds the files and issues that relate to the web platform.
```

Comment:
Somehow it feels weird reading out these two sentences that follow each other:

> This repository holds the files and issues shared among all platforms.
> This repository holds the files and issues that relate to the web platform.

Could we somehow merge the two statements or rephrase so that it doesn't sound confusing?